lib/screens/v2/widget_instance/evergreen_content.ex

```diff
@@ -31,12 +31,6 @@ defmodule Screens.V2.WidgetInstance.EvergreenContent do
 
   def priority(%__MODULE__{} = instance), do: instance.priority
 
-  def serialize(%__MODULE__{
-        screen: %Screen{app_params: %Triptych{show_identifiers: show_identifiers}},
-        asset_url: asset_url
-      }),
-      do: %{asset_url: asset_url, show_identifiers: show_identifiers}
```

Comment:
The `show_identifiers` field doesn't seem to be used aside from Triptychs. Can we remove it from this module's struct+type outright?